conda/common/io.py

```diff
@@ -463,6 +469,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                 sys.stdout.flush()
 
 
+@deprecated(
+    "25.3",
+    "25.9",
+    addendum="Use `conda.reporters.get_progress_bar` instead.",
+)
```

Comment:
Forgot to add this deprecation warning for my progress bar pull request (#14083)